src/helpers/cmds.js

```diff
@@ -26,6 +29,47 @@ const addPositional = (args, action) => {
   }
 }
 
+const statuses = {
+  running: `🟢`,
+  success: `✅`,
+  error: `❌`,
+  cancelled: `❎`
+}
+
+const titleState = {
+  cliName: null,
+  packageManagerName: null,
+  action: null,
+  status: null
+}
+
+export const updateTerminalTitle = (status) => {
+  if (!status) {
+    throw new Error(`Terminal title status must be provided.`)
+  }
+
+  titleState.status = status
+
+  if (!titleState.cliName || !titleState.packageManagerName || !titleState.action) {
+    throw new Error(`Terminal title must be initialized before updating it.`)
+  }
+
+  setTitle(`${titleState.status} ${titleState.cliName}: ${titleState.packageManagerName} ${titleState.action}`)
```

Comment:
I'm used to using prettier but I don't see a prettierrc, how should I format?

Reply:
I'm only using eslint on this project with these configurations on `.vscode\settings.json` file.

```json
"[javascript][typescript]": {
    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
    "editor.formatOnSave": false,
    "editor.codeActionsOnSave": [
      "source.formatDocument",
      "source.fixAll.eslint"
    ],
  },
```

I think you need to have installed the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension to autoformat the code on save.